Other/MergeShops/OwlRabbitFoxMerge.cs

```diff
@@ -0,0 +1,94 @@
+tags: owl, rabbit, fox, merge, rivensylth, wise, suit, wings
+*/
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreFarms.cs
+//cs_include Scripts/CoreAdvanced.cs
+//cs_include Scripts/CoreDailies.cs
+//cs_include Scripts/CoreStory.cs
+//cs_include Scripts/Story/QueenofMonsters/Extra/BrightOak.cs
+using Skua.Core.Interfaces;
+using Skua.Core.Models.Items;
+using Skua.Core.Options;
+
+public class OwlRabbitFoxMerge
+{
+    private IScriptInterface Bot => IScriptInterface.Instance;
+    private CoreBots Core => CoreBots.Instance;
+    private CoreFarms Farm = new();
+    private CoreAdvanced Adv = new();
+    private static CoreAdvanced sAdv = new();
+
+    private BrightOak BO = new();
+
+    public List<IOption> Generic = sAdv.MergeOptions;
+    public string[] MultiOptions = { "Generic", "Select" };
+    public string OptionsStorage = sAdv.OptionsStorage;
+    // [Can Change] This should only be changed by the author.
+    // If true, it will not stop the script if the default case triggers and the user chose to only get mats
+    private bool dontStopMissingIng = false;
+
+    public void ScriptMain(IScriptInterface Bot)
+    {
+        Core.BankingBlackList.AddRange(new[] { "Stories of the Forest"});
+        Core.SetOptions();
+
+        BuyAllMerge();
+        Core.SetOptions(false);
+    }
+
+    public void BuyAllMerge(string? buyOnlyThis = null, mergeOptionsEnum? buyMode = null)
+    {
+        BO.Ælfred();
+        //Only edit the map and shopID here
+        Adv.StartBuyAllMerge("rivensylth", 1180, findIngredients, buyOnlyThis, buyMode: buyMode);
+        Adv.StartBuyAllMerge("rivensylth", 1179, findIngredients, buyOnlyThis, buyMode: buyMode);
+        Adv.StartBuyAllMerge("rivensylth", 1178, findIngredients, buyOnlyThis, buyMode: buyMode);
+
+        #region Dont edit this part
+        void findIngredients()
+        {
+            ItemBase req = Adv.externalItem;
+            int quant = Adv.externalQuant;
+            int currentQuant = req.Temp ? Bot.TempInv.GetQuantity(req.Name) : Bot.Inventory.GetQuantity(req.Name);
+            if (req == null)
+            {
+                Core.Logger("req is NULL");
+                return;
+            }
+
+            switch (req.Name)
+            {
+                default:
+                    bool shouldStop = Adv.matsOnly ? !dontStopMissingIng : true;
+                    Core.Logger($"The bot hasn't been taught how to get {req.Name}." + (shouldStop ? " Please report the issue." : " Skipping"), messageBox: shouldStop, stopBot: shouldStop);
+                    break;
+                #endregion
+
+                case "Stories of the Forest":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.RegisterQuests(4645);
+                        Core.HuntMonster("rivensylth", "Rivensylth Spider", req.Name, quant, true, false);

```

Comment:
you'll need the while line here
`                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
{
huntline
]
`

since the reward comes from a quest?
if the item is strictly from a mob u don't need the registerquest